Add Searc,SearchCardBoard, Fix AppNavigation

biocle8339 opened the pull request.

biocle8339 committed:
ADD multiple features
-ADD Search, SearchCardBoard
-FIX AppNavigation

biocle8339 committed:
Merge branch 'develop' into features/static/Search

biocle8339 committed:
MERGE with current develop

biocle8339 committed:
REFACTOR Main, Search, TabBar, Header, SearchCardBoard

orange-brother commented on components/shared/Header/Header.js:
```diff
@@ -17,7 +17,7 @@ const Header = ({ navigation }) => {
       <TouchableOpacity
         style={styles.button}
-        onPress={() => navigation.navigate("config")}
+        onPress={() => navigation.navigate("Config")}
```
별건 아닌데 지금 screen 이름이 Configuration으로 되어 있어요!

orange-brother commented on constants/colors.js:
```diff
@@ -12,12 +12,20 @@ const TAG_LIGHT_BLUE = "#65AAEA";
 const TAG_BLACK = "#3C3A36";
 
 const PURE_WHITE = "#FFFFFF";
+const PURE_BLACK = "#000000";
```
그때 리뷰 받고 PRIMARY_WHITE = "#FFFFFF"; 추가해놨습니다. 그리고 PURE_BLACK 이 어디다 사용되는지 모르겠는데 PRIMARY_BLACK 이용해주세요 가독성 문제때문에 ㅠ

biocle8339 committed:
REFACTOR Multi features:
-REFACTOR Main, Header, TabBar
-REFACTOR Search, SearchCardBoard, SearchCard, BeerCard

orange-brother commented on components/SearchCardBoard/SearchCard/styles.js:
```diff
@@ -0,0 +1,55 @@
+    paddingHorizontal: 17,
+    paddingVertical: 10,
+    borderRadius: 8,
+    backgroundColor,
```
backgroundColor 확인 부탁드려요

orange-brother commented on components/SearchCardBoard/SearchCardBoard.js:
```diff
@@ -0,0 +1,65 @@
+
+  return (
+    <View style={styles.container}>
+      {isLoading ? <Text>Loading...</Text> : null}
```
지금 중요한건 아닌데 Loading 페이지 만들어놨어요!

biocle8339 committed:
FIX Header, Main, SearchCardBoard
